com.unity.multiplayer.mlapi/Runtime/Messaging/CustomMessageManager.cs

```diff
@@ -58,7 +58,8 @@ public void SendUnnamedMessage(List<ulong> clientIds, NetworkBuffer buffer, Netw
             {
                 using (var nonNullContext = (InternalCommandContext) context)
                 {
-                    nonNullContext.NetworkWriter.WriteBytes(buffer.GetBuffer(), buffer.Position);
+                    buffer.Position = 0;
```

Comment:
Is there any worry about this changing the Position value from what it was when it was passed in?

Reply:
I don't think so. This is a user-provided Stream, and my interpretation is that it's handed off to us to do whatever we please.

TBH, it might have as well been a byte array, since I don't think actually streaming this stuff would be something we'd want to do (ie: from disk or network or whatever). I don't expect these payloads to be considerable in size, but then again, I'm not a game developer or even a netcode developer :)

Reply:
Well, the serialization RFC gets rid of this whole custom message functionality (replaced with a much better, more flexible version of it) so I won't worry too much about it regardless.